packages/workflow/src/features/registration/fhir/fhir-bundle-modifier.ts

```diff
@@ -582,19 +582,19 @@ export function updatePatientIdentifierWithRN(
   return sectionCodes.map((sectionCode) => {
     const sectionEntry = getSectionEntryBySectionCode(composition, sectionCode)
     const patientId = urlReferenceToUUID(sectionEntry.reference as URLReference)
-    const patient = getResourceFromBundleById<Patient>(record, patientId)
+    const patientEntry = getResourceFromBundleById<Patient>(record, patientId)
```

Comment:
Doesn't it make more sense to keep it named as "patient" as we are dealing with the resource directly and not the entry